core/kazoo_data/src/kz_datamgr.erl

```diff
@@ -110,7 +110,7 @@
 %%--------------------------------------------------------------------
 -spec update_doc_from_file(ne_binary(), atom(), nonempty_string() | ne_binary()) ->
                                   {'ok', kz_json:object()} |
-                                  data_error().
+                                  kazoo_data:data_error().
```

Comment:
@lazedo I think we need something exported so calling modules don't need to use `{'error', any()}`. It could be `kz_datamgr:data_error()` or we could make specific ones for `open_doc` vs `get_results` if desired.